src/cli.cpp

```diff
@@ -219,7 +231,7 @@ void CLI::printID(DWORD *pid)
 }
 
 void CLI::printKey(char *pk) {
-    assert(strlen(pk) == 25);
+    assert(strlen(pk) >= 25);
```

Comment:
Keep the equals assert, we don't accept non-stripped keys here

Reply:
This assert was adjusted as "strlen" will often return above 25 for a filled in stack variable without a null terminator. `BINK1998::Verify` expects a 25-character input key buffer and will fail to compile when a null terminator is added to the `char product_key[PK_LENGTH]{};` variable that I run stripKey into. I'm not sure of any cleaner solution to this, other than adjusting the assert (which isn't ideal...)